### ISSUE TITLE
Hate Crime redirect on Landing Page needs external site modal

### ISSUE THREAD

**Issue by blaughman** [usdoj-crt/crt-portal-management#1189]
## Summary

Any link to a non fed site requires the exit popup.

## Detail
Clicking on the National Human Trafficking Hotline link goes directly to the site without a popup indicating you are going to an external site.

### Steps To Reproduce
1. Clicking this link
![image](https://user-images.githubusercontent.com/52507395/152004820-7af1d75e-cbee-425e-bcac-191a68fee999.png)
2. Goes directly to the site without a popup.

If it's relevant, add browser or operating system where the error occurs.

### Expected Behavior
1. Clicking this link
![image](https://user-images.githubusercontent.com/52507395/152004923-6d3d9126-b9bd-413e-a5cf-c732236fba54.png)
2. Should populate this modal
![image](https://user-images.githubusercontent.com/52507395/152005114-6d853157-85df-4252-ad8a-d62e3c33cfc6.png)

## Suggested Actions
Populate the popup modal when the link is clicked

## Acceptance Criteria
1. Given I am on the civilrights.justice.gov landing page
when I am researching the 'Victim of Human Trafficking' text
AND
I click the link to go to the 'National Human Trafficking Hotline' site
then I see a popup modal indicating I am going to an external (non fed) site

**Comment by blaughman** [comment]
Per discussion in kickoff meeting, we need to check with Terri regarding new USWDS guidance and its' impact on accessibility.

**Comment by louh** [comment]
Right, this is a great opportunity to bring attention to [current USWDS guidance on external links](https://designsystem.digital.gov/components/link/):

> **Don’t roadblock external links with a modal window or dialog box.** Allow users to follow external links without taking a separate action to acknowledge leaving your site.

This is not our current implementation strategy. However, it is also not necessarily as easy as simply removing the modal in all situations. To comply with OMB policies, we must:

1. **Clearly identify external links.** Note that OMB does not require that the identification take any specific format, only that "agencies should choose the best approach to identify external links to users in a way that minimizes the impact on the usability of their websites and digital services." In other words, if a modal impacts usability, then we can adopt a different method of identifying external links. [The USWDS recommends using plain-language content in the link text, like so](https://designsystem.digital.gov/components/link/#what-you-should-do), or as [a parenthetical tag, like so [digital.gov]](https://designsystem.digital.gov/components/link/#what-you-should-do), along with an external link icon.
2. **Provide required notification of non-endorsement of non-federal sites.** Again, because the OMB does not specifically require that notification be in a modal, the USWDS recommends adding this language to a separate "policies and notices" page on the site. There are examples at [designsystem.digital.gov](https://designsystem.digital.gov/about/website-policies-notices/#linking-policy) and [usa.gov](https://www.usa.gov/linking-policy). Our current footer links to ["Legal policies and Disclaimers"](https://www.justice.gov/legalpolicies) hosted on the main justice.gov website which may also adequately serve this purpose.

In this particular case, "Get from the National Human Trafficking Hotline" clearly indicates a user is going to an external organization in plain language, plus we are using an external link indicator.

As for the required notification of non-endorsement, this is either covered by the justice.gov legal policy page _or_ we may need one specifically for the CRT portal, matching our layout and branding.

I do agree that we should do one more check-in on accessibility impacts.

**Comment by randyabramson** [comment]
@blaughman - I feel like if we write very straight forward link language, we can move this forward. I am not sure this link would trigger any accessibility issues. I feel like we can move this forward and have Terri check it later or even on DEV

**Comment by blaughman** [comment]
@randyabramson I'm fine with this approach.

**Comment by blaughman** [comment]
Link operates as intended.  Moving to UAT.

**Comment by blaughman** [comment]
Link operates as intended in Staging.  Ready for Production.

**Comment by blaughman** [comment]
Functions as intended in Production.  Closing this card.
